doc/topics/cloud/vmware.rst

```diff
@@ -283,6 +283,9 @@ Set up an initial profile at ``/etc/salt/cloud.profiles`` or
         thin_provision
             Specifies whether the disk should be thin provisioned or not. Default is ``thin_provision: False``.
             .. versionadded:: 2016.3.0
+        eagerly_scrub
+            Specifies whether the disk should be rewrite with zeros during thick provisioning or not.
+            Default is ``eagerly_scrub: False``.
```

Comment:
Can you add a `.. versionadded:: Oxygen` tag here?

Reply:
i added the version tag

Reply:
thanks!